src/hosted_db/sqlite.rs

```diff
@@ -36,3 +47,35 @@ pub fn run_sqlite_query(path: &PathBuf, query: &str) -> Result<QueryResult, AybE
         rows: results,
     })
 }
+
+pub async fn run_sqlite_query(
+    path: &PathBuf,
+    query: &str,
+    isolation: &Option<AybConfigIsolation>,
+) -> Result<QueryResult, AybError> {
+    match isolation {
+        Some(isolation) => {
+            let result = run_in_sandbox(Path::new(&isolation.nsjail_path), path, query).await?;
+
+            if result.stderr.len() > 0 {
```

Comment:
```suggestion
            if !result.stderr.is_empty() {
```

Reply:
Nice! Thank you!

(I noticed `cargo clippy` also caught this, and realized clippy isn't breaking the build, so I created #250)